recipes/crow/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(cpp-jwt REQUIRED)
```

Comment:
needs to be changed

Reply:
good catch, I've just used another project as template and forgot to update the example. Just updated the test